zerver/actions/users.py

```diff
@@ -609,7 +610,7 @@ def send_stream_events_for_role_update(
 
 @transaction.atomic(savepoint=False)
 def do_change_user_role(
-    user_profile: UserProfile, value: int, *, acting_user: UserProfile | None
+    user_profile: UserProfile, value: int, *, acting_user: UserProfile | None, notify: bool = False
```

Comment:
I wonder if these should be `notify: bool`, without the `= False` -- i.e., require code that is controlling this to be explicit about whether it's notifying, so that code that forgets to pass `notify=True` is visibly wrong.

What's the motivation for how you've written this -- is it just to avoid updating all the tests that call this?